### PR TITLE
add update repository url command

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by yanan** [comment]
ft retest

File: tools/plugins/com.liferay.ide.upgrade.commands.core/src/com/liferay/ide/upgrade/commands/core/internal/dependencies/UpdateRepositoryURLCommand.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ServiceScope;
+
+/**
+ * @author Terry Jia
+ */
+@Component(
+	property = "id=" + UpdateRepositoryURLCommandKeys.ID, scope = ServiceScope.PROTOTYPE, service = UpgradeCommand.class
+)
+public class UpdateRepositoryURLCommand implements UpgradeCommand {
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.upgrade.commands.core/src/com/liferay/ide/upgrade/commands/core/internal/dependencies/UpdateRepositoryURLCommand.java]: Hey @jtydhr88  can you use the new UpgradePreview feature for this one as well?

File: tools/plugins/com.liferay.ide.upgrade.commands.ui/src/com/liferay/ide/upgrade/commands/ui/internal/MessagePromptImpl.java [hunk 2]
```diff
@@ -26,6 +26,11 @@
 @Component(service = MessagePrompt.class)
 public class MessagePromptImpl implements MessagePrompt {
 
+	@Override
+	public void postInfo(String title, String message) {
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.upgrade.commands.ui/src/com/liferay/ide/upgrade/commands/ui/internal/MessagePromptImpl.java]:
```suggestion
	public void promptInfo(String title, String message) {
```

File: tools/plugins/com.liferay.ide.upgrade.commands.ui/src/com/liferay/ide/upgrade/commands/ui/internal/MessagePromptImpl.java [hunk 3]
```diff
@@ -26,6 +26,11 @@
...
+		UIUtil.postInfo(title, message);
+	}
+
 	@Override
 	public boolean prompt(String title, String message) {
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.upgrade.commands.ui/src/com/liferay/ide/upgrade/commands/ui/internal/MessagePromptImpl.java]: Lets make this method `promptQuestion`

File: tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/internal/07-liferay-code-upgrade-plan.markdown [hunk 4]
```diff
@@ -63,7 +63,7 @@ Here are the steps for upgrading your development environment and custom plugins
         <div class="description">Upgrade your build dependencies.</div>
     </li>
     <ol>
-        <li>
+        <li commandId="update_repository_url">
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/internal/07-liferay-code-upgrade-plan.markdown]: Can you update the markdown over on my branch here?  https://github.com/gamerson/liferay-docs/tree/upgradePlanner as well?

**Comment by gamerson** [comment]
also @jtydhr88  can you rebase on master?  thanks!

**Pull request closed without merging**